commands/clean.js

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env node
+const { log, runCommandGroup } = require('../lib.js');
+
+function clean(projects, options) {
```

Comment:
several of these are now very similar.. I wonder if it makes sense to deduplicate them by parametrizing a factory function, e.g. `clean: generateCommandGroupRunner('cleanCommands')`

Reply:
Yeah makes sense, though I don't know yet if they're going to diverge. They might (`setup` did). Leaving a note for now!